### DISCUSSION TITLE
Uploading image

### DISCUSSION THREAD

**Discussion by proxylady** [phcode-dev/phoenix#1106] [Q&A]
Hi, today I started my education about HTML and CSS so I'm using phoenix as a tool to learn. In one exercise we needed to upload image from our computer to image folder on ph and I couldn't find a way to do it. I'm aware that this is basic of the basics but please don't hold it against me :D just a beginner that wants to learn.
Thank you!

**Comment by abose** [comment]
Phoenix works with local files in your machine and there is no upload concept in phcode. Just open any folder in your pc and it should pick up any Images in the folder. Copy files to the folder using finder/file explorer and press refresh file tree in phcode to pick up any images recently added to your project directory.

![image](https://github.com/phcode-dev/phoenix/assets/5336369/7b9dbd4a-49b1-4463-9578-14b6e6c8d5c3)
